modules/tinymce/src/plugins/save/main/ts/core/Actions.ts

```diff
@@ -37,7 +36,9 @@ const save = function (editor) {
   if (formObj) {
     editor.setDirty(false);
 
-    if (!formObj.onsubmit || formObj.onsubmit()) {
+    // TODO: this is probably broken, as an event should be passed to `onsubmit`
```

Comment:
Is there a matched Jira?

Reply:
@metricjs did log one, I'll see if I can dig it up.